crates/syn-solidity/src/attribute/function.rs

```diff
@@ -155,7 +148,6 @@ impl fmt::Debug for FunctionAttribute {
             Self::Visibility(visibility) => f.debug_tuple("Visibility").field(visibility).finish(),
             Self::Mutability(mutability) => f.debug_tuple("Mutability").field(mutability).finish(),
             Self::Virtual(_) => f.write_str("Virtual"),
-            Self::Immutable(_) => f.write_str("immutable"),
```

Comment:
drve-by nit: uncaptialize `"Virtual"` on the line above

Reply:
It's debug fmt